lib/health.js

```diff
@@ -170,6 +170,53 @@ const cmdExec = (cmd, callback) => {
   });
 };
 
+const getSmartStatusOfDrive = (driveName, callback) => {
+  const cmd = `smartctl -H /dev/${driveName}`;
+  api.cp.exec(cmd, (error, stdout, stderr) => {
+    if (error) {
+      callback(error);
+      return;
+    }
+
+    if (stderr) {
+      callback(stderr);
+      return;
+    }
+    const status = stdout.toString().split(':').pop().trim();
+    callback(null, status === 'PASSED');
+  });
+};
+
+
+health.getSmartStatus = callback => {
+  const cmd = 'lsblk -d -J';
+  const drivesStatus = {};
+  api.cp.exec(cmd, (error, stdout, stderr) => {
+    if (error) {
+      impress.log.error(stderr);
+      callback(error);
+      return;
+    }
+
+    const funcs = JSON.parse(stdout).blockdevices.map(drive =>
+      next => getSmartStatusOfDrive(drive.name, (error, status) => {
+        if (error) {
+          impress.log.error(stderr);
```

Comment:
Why do you log the `stderr` obtained when running `lsblk -d -J` when you get an error from `getSmarStatusOfDrive()`?
It is now possible for it to be logged more than once.